packages/babel-plugin-transform-typescript/src/index.js

```diff
@@ -279,6 +279,9 @@ export default declare((api, { jsxPragma = "React" }) => {
       JSXElement() {
         sourceFileHasJsx = true;
       },
+      JSXFragment() {
+        sourceFileHasJsx = true;
+      },
```

Comment:
Actually we could just use `JSX() { sourceFileHasJsx = true }` which covers both

Reply:
Yeah or "JSXElement|JSXFragment(){}", but not sure I care

Reply:
I don't really care either but I have a preference for the jsx since it's smaller.